### DISCUSSION TITLE
plugin not accepted on github

### DISCUSSION THREAD

**Discussion by jphconstantin** [mmistakes/minimal-mistakes#2588] [Q&A] [answered]
Hello,
I added a plugin (which run correctly locally with minimal-mistakes) and github seems to not accept it. The author confirmed it. The plugin is jekyll-exif-data.
Does exist a work around ?

**Comment by iBug** [comment, marked as answer]
<https://docs.github.com/en/github/working-with-github-pages/about-github-pages-and-jekyll#plugins>

If you want to use unsupported plugins, you can instead build your site with a CI service like Travis CI. I have written an article years back which you can refer to.

<https://ibug.io/p/4>
